docs/gen-rnd-pan-component.md

```diff
@@ -0,0 +1,37 @@
+# gen_rnd_pan()
+
+Implemented in Percona Server for MySQL 8.0.33-26.
+
+Generates an Primary Account Number (PAN) for a payment card.
+
+!!! important
+
+    Generating the PAN should only be used for testing. The function does not check if the generated value is a legitimate primary account number. If you must publish the result, consider using [`mask_pan`](./mask-pan-component.md) or [`mask_pan_relaxed()`](./mask-pan-relaxed-component.md) to obfuscate the result.
+
+## Parameters
+
+| Parameter | Optional | Description | Type |
+| --- | --- | --- | --- |
+| size | Yes | Number of characters | Integer |
```

Comment:
Our implementation doesn't have a size parameter, instead we generate a random valid-looking credit card number.

Reply:
Removed.